src/converter/renderers/render-aem.js

```diff
@@ -27,16 +122,15 @@ function transformHTML(htmlString, aemAuthorUrl, path) {
     if (uri.startsWith('/') && !isAbsoluteURL(uri))
       el.setAttribute('content', relativeToAbsolute(uri, aemAuthorUrl));
   });
-  // added no indexing rule for author bio pages
+  // no indexing rule for author bio pages
   if (path.includes('/articles/authors')) {
-    const robotsMeta = document.createElement('meta');
-    robotsMeta.setAttribute('name', 'robots');
-    robotsMeta.setAttribute(
-      'content',
-      'NOINDEX, NOFOLLOW, NOARCHIVE, NOSNIPPET',
-    );
-    document.head.appendChild(robotsMeta);
+    setMetadata(document, 'robots', 'NOINDEX, NOFOLLOW, NOARCHIVE, NOSNIPPET');
   }
+  // Update page metadata for AEM Article Pages
+  if (path.includes('/articles/') && !path.includes('/articles/authors/')) {
```

Comment:
We may not want to apply this to article landing page. We could also use the "theme" meta to determine if the given page is an article.

Reply:
We don't get the "theme" information in converter.